applications/GeoMechanicsApplication/custom_conditions/T_normal_flux_condition.cpp

```diff
@@ -71,12 +68,11 @@ void GeoTNormalFluxCondition<TDim, TNumNodes>::CalculateRHS(Vector&            r
         // Interpolation of nodal normal flux to integration point normal flux.
         auto normal_flux_on_integration_point = MathUtils<>::Dot(N, normal_flux_vector);
 
-        auto weighting_integration_coefficient =
-            ConditionUtilities::CalculateIntegrationCoefficient<TDim, TNumNodes>(
-                j_container[integration_point], r_integration_points[integration_point].Weight());
+        auto weighting_integration_coefficient = ConditionUtilities::CalculateIntegrationCoefficient(
```

Comment:
We can make this variable `const`:
```suggestion
        const auto weighting_integration_coefficient = ConditionUtilities::CalculateIntegrationCoefficient(
```

Reply:
done and renamed to the usual integration_coefficient.